glide.yaml

```diff
@@ -110,7 +110,8 @@ import:
 - package: github.com/openshift/library-go
   version: master
 - package: github.com/openshift/apiserver-library-go
-  version: master
+# temporary pin to avoid the kube 1.17 rebase
+  version: 7c5911e93780807f2082f34b259dfaf97a108222
```

Comment:
@mfojtik @deads2k @sttts for now this hacks around the fact that origin is not compatible w/ master apiserver-library-go (until origin is rebased on kube 1.17).

This PR isn't really urgent, so if you'd rather just wait until origin is rebased that's fine, just let me know either way.

Reply:
sounds like https://github.com/openshift/origin/pull/24286 will resolve this, i'll wait for that.